testing: add tiny framework for testing rustic's compat with latest restic (#1303)
rustic-rs/rustic: tests/repository-fixtures/README.md

# Repository Fixtures

This directory contains fixtures for testing the `rustic` and `restic`
repositories.

> The `rustic` repository is a simple repository that is used to test the `rustic`
> library. The `restic` repository is a repository created by the `restic` backup

Author: ```suggestion
The `rustic` repository is a repository that is used to test the `rustic`
binary. The `restic` repository is a repository created by the `restic` backup
```

> tool. It is used to test `rustic` against a repository created with restic.

Author: ```suggestion
tool.
```

# Repository Fixtures

This directory contains fixtures for testing the `rustic` and `restic`
repositories.

The `rustic` repository is a simple repository that is used to test the `rustic`
library. The `restic` repository is a repository created by the `restic` backup

> tool. It is used to test the `rustic` against a repository created with restic.

Author: ```suggestion
tool. It is used to test `rustic` against a repository created with restic.
```

The latter is used to ensure that `rustic` can read and write to a repository
created by `restic`.

> TODO: We should create a `restic` repository live during our tests. This way we
> can ensure that the repository is always compatible with the latest version of
> `restic`.
> 

Author: ```suggestion
```
